ziti/ziti.go

```diff
@@ -1285,9 +1285,7 @@ func (context *ContextImpl) getEdgeRouterConn(session *rest_model.SessionDetail,
 	var bestER edge.RouterConn
 	var unconnected []*rest_model.SessionEdgeRouter
 	for _, edgeRouter := range session.EdgeRouters {
-		for proto, addr := range edgeRouter.SupportedProtocols {
-			addr = strings.Replace(addr, "://", ":", 1)
-			edgeRouter.SupportedProtocols[proto] = addr
```

Comment:
IIRC this was for backward compatibility with older controllers. I assume it's removal means we have aged out of those controllers sufficiently.